_rules/auto-update-text-efbfc7.md

```diff
@@ -22,16 +22,16 @@ acknowledgements:
 
 The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
 
-- the `innerText` property of the [element][html element] is updated with a determinable frequency;
-- the [element][html element] does not have [children][child] whose `innerText` property is also updated;
-- the updating starts when the [readiness][document readiness] of the [HTML document][] the [element][html element] belongs to is equal to "complete";
+- the `innerText` property of the [element][html element] changes without the user [activating][activate] or [focusing][focus] any [element][html element] in the same [HTML document][];
```

Comment:
I like the way this goes 😄 

* I'm not entirely sure this takes care of the delay between activation and change (if I click a button and the page changes 2h later, is it OK?) Maybe add a condition like "the window has [Transient activation](https://html.spec.whatwg.org/#transient-activation)" to put a reasonable time limit which is (AFAICT) UA dependant, hence we're not the ones deciding what is "reacting to user input".
* Can I unfocus an element without focusing another one? If so, that might be a user activation which can change page and would be good for WCAG but not for this rule. Maybe refer to [Activation triggering input event](https://html.spec.whatwg.org/#activation-triggering-input-event) for an "official" list of what is user interaction.
* Similar question with global events that are not really related to any element but can still change stuff.

Also, wondering whether there should be an assumption on only looking at activations enabled by content (and not, say, browser shortcuts or extensions), _à la_ https://github.com/act-rules/act-rules.github.io/pull/873/files#diff-080b736abdfda182cd29c13427ee2aecR34

Reply:
You're focusing on changes that result from user actions. The rule applies to changes that do not result from user actions. If the user did something that triggered an update that only happens 2 hours in the future then that is not applicable (I'm not saying it's okay, but I don't think we can go down that path without compromising the objectivity so I argue that we should stick with an applicability that does not consider "delayed updates").

Good comment about the assumption. I'll add it.

Reply:
Hmm, so you're saying that a delayed activation is considered as a user activation and that will make the rule not fail but the SC fail, which is OK.

I think you have a point…

I was suggesting to use the Transient activation state as it is an objective and unambiguous state that UA consider are "close enough to user action that we can still do stuff". But it might indeed get too hairy.

Hum, no, you're right, we can't really put a delay because something like Inapplicable Example 2 (which is OK) has updates that happen as long after the click as we want (they never stop).

---
I still think there is an issue on what is defined as "user interaction".
Typically, Inapplicable Example 2 is good because the updates only start after I click (and never stop).

What if there is a page with a global event listener (not linked to any element). When I press `ctrl+s`, the update start. This is not the user activating or focusing any element, so the rule is applicable. There is no component to stop the update, so the rule fail. But SC 2.2.2 does not fail because the update does not "start automatically". Failing the rule without failing the SC is bad…